src/Common/src/Interop/Unix/libssl/Interop.OpenSsl.cs

```diff
@@ -414,16 +404,22 @@ private static SslException CreateSslException(IntPtr sslPtr, string message, in
 
         private sealed class SslException : Exception
         {
-            public SslException(string message)
-                : base(message + ": " + Marshal.PtrToStringAnsi(libssl.ERR_reason_error_string(libssl.ERR_get_error())))
+            private string message;
+            public override string Message { get { return message; } }
```

Comment:
Why store the message like this?  How about just having a helper that produces the message given the ctor arguments and passes the result of that down to the base class?